third_party/forked/afex/hystrix-go/hystrix/circuit.go

```diff
@@ -35,6 +35,17 @@ func init() {
 	circuitBreakers = make(map[string]*CircuitBreaker)
 }
 
+// IsCircuitBreakerOpen returns whether a circuitBreaker is open for an interface
+func IsCircuitBreakerOpen(name string) bool {
+	circuitBreakersMutex.Lock()
+	defer circuitBreakersMutex.Unlock()
+	if c, ok := circuitBreakers[name]; ok {
+		return c.IsOpen()
+	} else {
+		return false
+	}
```

Comment:
else return err 
ErrCBNotexist= errors.New()

Reply:
Have changed it.